src/internet_identity/tests/integration/openid.rs

```diff
@@ -446,14 +653,42 @@ fn mock_google_certs_response(env: &PocketIc) {
         }
 
         if attempts >= MAX_ATTEMPTS {
-            panic!("No Google cert requests found after {MAX_ATTEMPTS} attempts");
+            panic!("No cert requests found for URL '{url}' after {MAX_ATTEMPTS} attempts");
         }
     }
 }
 
-pub fn openid_test_data() -> (String, [u8; 32], Claims, u64, Principal, AuthnMethodData) {
-    // This JWT is for testing purposes, it's already been expired before this commit has been made,
-    // additionally the audience of this JWT is a test Google client registration, not production.
+/**
+ * Explanation of the fields used for the test data:
+ * - JWT: the JWT received by the openID provider.
+ * - salt: the salt used to create the JWT from before.
+ * - test_time: the `iat` (issued at) field from the JWT
+ * - test_principal: the principal of the identity used the link the OpenID account.
+ * - test_pubkey: the public key of the credential used to sign in with the identity from `test_principal`.
+ *   Not the public key of the principal of the identity. You don't get it with connection.identity.getPublicKey().
+ *
+ * How to get the test data:
```

Comment:
I wonder if we could automate the generation of this via `playwrite`? 

No changes required for this PR